bdai_ros2_wrappers/bdai_ros2_wrappers/__init__.py

```diff
@@ -4,7 +4,13 @@
 import pkgutil
 import sys
 
+from bdai.deprecation import deprecated
```

Comment:
think we can't use this import here since ros_utilities can't depend on bdai

Reply:
ya ive just realized that but the built in version isnt printing properly when i run a test
changing this to a draft in the meantime